mobile/lib/modules/home/views/home_page.dart

```diff
@@ -35,6 +36,7 @@ class HomePage extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final multiselectEnabled = ref.watch(multiselectProvider.notifier);
     final selectionEnabledHook = useState(false);
+    final selectionMode = useState(AssetState.remote);
```

Comment:
Hmm, maybe I'd call this something else like `selectionAssetState`?

Reply:
Sounds better. Made the change and refactored the method to a variable in the process.